OPS-54: Add/Update AA

motionaviation opened the pull request.

motionaviation committed:
Add/Update AA

motionaviation committed:
Updated Holding Points

motionaviation committed:
OPS-54: Updated Holding Positions and added Divergent Departures

rbywater commented on docs/aerodromes/Class-C/NZAA/index.md:
```diff
@@ -4,113 +4,89 @@
+## Layout
 
-Auckland Tower is resposible for VFR aircraft in the North or South sector (as depicted in the Auckland VFR Arrivals/Departures page in the AIP) ensuring they are separated from IFR traffic whilst on long final for an instrument approach. This does not include aircraft on some RNP approaches.
+Auckland Airport has one runway 05R/23L, and it home to around 100 gates located around the airport. Auckland is New Zealand's largest international airport serving over 57 airports with 24 airlines. 
```
```suggestion
Auckland Airport has one runway, 05R/23L, and is home to around 100 gates located around the airport. Auckland is New Zealand's largest international airport serving over 57 airports with 24 airlines. 
```

rbywater commented on docs/aerodromes/Class-C/NZAA/index.md:
```diff
@@ -4,113 +4,89 @@
+### Domestic 
 
-Arrivals on an RNP approach, not straight-in - at ATAMA or NIPAS (RWY05R), ESNAX, ESMON, TEGIK, or TUSLO (RWY23L).
+Aircraft pushing back from gates 20-22 normally will push back onto D1/D2 depending on other movements or incases onto B. Aircraft on 24 can be pushed back to whereever the controller sees suitable at the time. Aircraft on 28-33 shall generally be pushed facing the direction of the active runway, although given the layout that can change. 
```
```suggestion
Aircraft pushing back from gates 20-22 normally will push back onto D1/D2 depending on other movements or in some cases onto B. Aircraft on 24 can be pushed back to wherever the controller sees suitable at the time. Aircraft on 28-33 shall generally be pushed facing the direction of the active runway although, given the layout, that can change. 
```

rbywater commented on docs/aerodromes/Class-C/NZAA/index.md:
```diff
@@ -4,113 +4,89 @@
+### International 
 
-### Pushback & Movement Areas
+At international gates things are more simple, aircraft pushing off of any gate at international are given a tail facing direction and a start location. For example: "Push tail north start abeam stand 15". Aircraft tend to be towed forward but we will ignore this. 
```
```suggestion
At international gates things are simpler. Aircraft pushing off of any gate at international are given a tail facing direction and a start location. For example: "Push tail north start abeam stand 15". Aircraft tend to be towed forward but we will ignore this. 
```

rbywater commented on docs/aerodromes/Class-C/NZAA/index.md:
```diff
@@ -4,113 +4,89 @@
+## Taxiing 
 
-Ground uses positive control to ensure that no other aircraft passes directly behind an aircraft pushing back, even when the aircraft concerned are not on the controlled movement area, this includes the C aprons, such as C5.
+With the way the taxiways are layed out at Auckland, controllers avoid using taxiway A where needed. When RWY 23L is active, domestic aircraft may be taxied where they need as they dont interat with rapid exits like A6/A8. In high traffic loads or to aliviate delays international aircraft are taxied down via B and B6 instead of all the way along A. 
```
```suggestion
With the way the taxiways are laid out at Auckland, controllers avoid using taxiway A where needed. When RWY 23L is active, domestic aircraft may be taxied where they need as they don't interact with rapid exits like A6/A8. In high traffic loads or to alleviate delays international aircraft are taxied down via B and B6 instead of all the way along A. 
```

rbywater commented on docs/aerodromes/Class-C/NZAA/index.md:
```diff
@@ -4,113 +4,89 @@
-### Transfer of Communication
-
-Once Auckland Ground has issued all necessary instructions to the pilot, they advise the pilot to monitor Auckland Tower on `118.700 MHz`.
+When RWY 05R is active any domestic aircraft or aircraft past domestic will be given taxi via B and L or aliviate the possiblilty of departures mixing with arrivals. Taxiway M may be utilized by contollers to get aircraft to A10 while there is an aircraft taxiing to A9. 
```
```suggestion
When RWY 05R is active any domestic aircraft or aircraft past domestic will be given taxi via B and L or alleviate the possibility of departures mixing with arrivals. Taxiway M may be utilized by controllers to get aircraft to A10 while there is an aircraft taxiing to A9. 
```

rbywater reviewed (commented):
Sorry was having a read through things and noticed a few possible changes. Hope you don't mind :)

motionaviation committed:
Update docs/aerodromes/Class-C/NZAA/index.md

Co-authored-by: Richard Bywater <richard@bywater.nz>

motionaviation committed:
Update docs/aerodromes/Class-C/NZAA/index.md

Co-authored-by: Richard Bywater <richard@bywater.nz>

motionaviation committed:
Update docs/aerodromes/Class-C/NZAA/index.md

Co-authored-by: Richard Bywater <richard@bywater.nz>

motionaviation committed:
Update docs/aerodromes/Class-C/NZAA/index.md

Co-authored-by: Richard Bywater <richard@bywater.nz>

motionaviation committed:
Update docs/aerodromes/Class-C/NZAA/index.md

Co-authored-by: Richard Bywater <richard@bywater.nz>

motionaviation committed:
OPS-54 Updated AA

Added Ground Movements Image etc.

motionaviation committed:
Highlight Taxiway Designators

mwr2410 committed:
small tweaks to some sections

- fixed image formatting
- fixed link formatting
- adjusted taxiway designators with `backtick`

dirtyformal committed:
Move file to main directory instead of subdirectory

dirtyformal committed:
Update Position Tables

dirtyformal committed:
Formatting, and 'shall' terminology

dirtyformal committed:
Addition SID info. Reformat headings.

dirtyformal committed:
Add Frequencies

mwr2410 committed:
Update abbreviations.md

add SMC - ADC - DCL - PDC abbreviations

mwr2410 committed:
Merge branch 'ops-54' of https://github.com/vatnz-dev/sops into ops-54

mwr2410 reviewed (approved):
as per linear
